dev/docker/kerberos-hive/hive-site1.xml

```diff
@@ -0,0 +1,77 @@
+<configuration>
+  <property>
+    <name>hive.server2.enable.doAs</name>
+    <value>true</value>
+    <description>Disable user impersonation for HiveServer2</description>
```

Comment:
enable or disable?

Reply:
It's a mistake that exists for a long time, I copied the content from the Hive docker file.

Reply:
I fixed the mistake for Kerberos Hive docker fille, as for Hive without Kerberos, I believe we need to use another PR to fix it or fix it by the way.

Reply:
for Hive without Kerberos can be fixed in another image updating